CHANGELOG.md

```diff
@@ -4,10 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-### Unreleased
+### [4.0.9] - 2023-01-03
```

Comment:
Should we keep the unreleased section and just make it blank?

Reply:
good call, added it back